Add English WNP for Firefox 105 [fix #12130]

craigcook opened the pull request:
## One-line summary

Adds English whatsnew page for Firefox 105.

## Issue / Bugzilla link

#12130 


## Testing

http://localhost:8000/en-US/firefox/105.0/whatsnew/

craigcook commented:
Note this is likely to cause some merge conflicts with https://github.com/mozilla/bedrock/pull/12090 so whichever one merges first, the other one will need a careful rebase.

alexgibson commented:
I just merged https://github.com/mozilla/bedrock/pull/12090 (sorry! 🙈) so this will need a rebase. Doesn't look too bad thankfully :)

alexgibson commented:
I'll review this whilst I'm here by way of an apology

alexgibson commented on bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html:
```diff
@@ -0,0 +1,62 @@
+
+    <div class="mzp-l-columns mzp-t-columns-three mzp-t-picto-center">
+      <div class="mzp-c-picto">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="" alt="216" class="mzp-c-picto-image">
```
NIt: looks like you need to swap the `height` and `alt` values around on these three images:
```suggestion
        <img src="{{ static('img/firefox/whatsnew/whatsnew105/privacy.png') }}" width="320" height="216" alt="" class="mzp-c-picto-image">
```

craigcook replied:
🤦

alexgibson commented on media/css/firefox/whatsnew/whatsnew-105-en.scss:
```diff
@@ -0,0 +1,59 @@
+}
+
+.wnp-main-img {
+    max-width: $content-md;
```
Issue (non-blocking): looks like this is preventing the image from resizing on smaller viewports?

alexgibson commented on bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx105-en.html:
```diff
@@ -0,0 +1,62 @@
+<section class="wnp-content-main mzp-l-content mzp-t-content-lg">
+  <h2 class="wnp-main-title">How to talk to your kids about online safety</h2>
+
+  <img src="{{ static('img/firefox/whatsnew/whatsnew105/tech-talk.png') }}" width="720" height="405" alt="The Tech Talk" class="wnp-main-img">
```
Suggestion (non-blocking): a responsive image would be a nice to have here, mainly so we're not serving people on low res monitors a @2x sized image?

alexgibson reviewed (changes requested):
A couple of tiny issues / nits, but otherwise this all looks good (after a rebase and minor shuffle of the unit tests).

craigcook commented:
Rebased and updated. I initially skipped responsive images because the high-res ones were already such small files and I don't know how many people still have low-res screens these days (other than me, because I haven't bought a new monitor in 15 years). But the new low-res images are even tinier still and I guess every little performance boost helps.

alexgibson reviewed (approved):
r+
